components/sensor/tsl2591.rst

```diff
@@ -126,7 +128,11 @@ For the TSL2591 device:
 - **update_interval** (*Optional*, :ref:`config-time`): The interval for checking the sensors.
   Defaults to ``60s``.
 - **power_save_mode** (*Optional*, boolean) Should the device be powered down between update intervals?
-  Defaults to ``False`` (*recommended*, see discussion below).
+  Defaults to ``True``
+- **device_factor** (*Optional*, float) The default is ``53.0``.
+  The device factor to be used as part of the lux equation for **caclulated_lux**.
+- **glass_attenuation_factor** (*Optional*, float) The default is ``7.7``.
+  The glass attenuation factor to be used as part of the lux equation for **caclulated_lux**.
```

Comment:
```suggestion
  The glass attenuation factor to be used as part of the lux equation for **calculated_lux**.
```

Reply:
I have learned in this exercise that I cannot faithfully type the word "calculated", nor the word "attenuated", without concentrating. I already fixed about a million of them. Thanks for catching these.